table/server/common/src/test/java/alluxio/table/common/udb/UdbFilterSpecTest.java

```diff
@@ -0,0 +1,258 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.table.common.udb;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThrows;
+import static org.junit.Assert.assertTrue;
+
+import alluxio.table.common.udb.UdbFilterSpec.Mode;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.regex.Pattern;
+
+public class UdbFilterSpecTest {
+  private UdbFilterSpec.Builder mBuilder;
```

Comment:
```suggestion
  private UdbFilterSpec.Builder mBuilder = new UdbFilterSpec.Builder();
```
and you don't need `before()`

better to keep code concise as long as readability is not reduced

Reply:
Fixed in 391b8b369413e0ba5c9528c2c0645cc54ee76932